23.2.2 added (#1)
friedcucumber/maui-theme-standard-controls: README.md

# DevExpress .NET MAUI Controls - Apply Themes to Standard Controls

> The DevExpress .NET MAUI Controls ships with 10 built-in color themes that are available in both dark and light variants. You can create styles for standard .NET MAUI Controls to apply our color themes to them. This repository demonstrates how to create styles and bind colors from our themes to a standard control's properties.

Reviewer: ```suggestion
DevExpress .NET MAUI Controls ship with 10 built-in color themes that are available in both dark and light variants. You can also apply these color themes to standard .NET MAUI Controls. This repository creates required styles and bind colors from DevExpress themes to properties of standard controls.
```
